util/election/runner_test.go

```diff
@@ -141,8 +142,9 @@ func TestElectionRunnerRun(t *testing.T) {
 			fakeTimeSource := util.NewFakeTimeSource(startTime)
 
 			el := test.election
-			tracker := election.NewMasterTracker([]int64{logID}, nil)
-			er := election.NewRunner(logID, &cfg, tracker, nil, el)
+			logString := strconv.FormatInt(logID, 10)
```

Comment:
Can you just do `const logID = "6962"` above?

Reply:
done